### PR TITLE
feat: export media [AR-2445][AR-3008]

### PR DESCRIPTION
**Pull request by MohammedMokresh**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2445" title="AR-2445" target="_blank">StoryAR-2445</a>  Export media from conversations
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues
Users can't share media with outer apps

### Solutions
Implement a way when a user long pressed on an asset they will have a share button to share it with other apps

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1416/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1416/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5644584)(5644584)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5644584)(5644584)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1416/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5646853)(5646853)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5646853)(5646853)-staging-compat.apk)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -110,6 +110,7 @@
     <string name="content_description_attachment_item">Attach new item to conversation</string>
     <string name="content_description_open_message_details">Open Message Details</string>
     <string name="content_description_copy_the_message">Copy the message</string>
+    <string name="content_description_share_the_file">Share the file</string>
```
Comment (gongracr) [inline app/src/main/res/values/strings.xml]: Maybe to be more precise update content description to 
```suggestion
    <string name="content_description_share_the_file">Share the selected file with other apps</string>
```

**Review by gongracr — APPROVED** [review]
LGTM 🍬

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-1416/8/) **failed**.

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-1416/9/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5671381)(5671381)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5671381)(5671381)-staging-compat.apk)

**Pull request merged**